### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by Ishadijcks**
Add basic routenavigation using the worldmap.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -132,7 +131,364 @@
             </div>
 
             <div class="page-item" style="background-color: green; min-height: 350px">
-                <h2>Map</h2>
+                <svg id="map" viewBox="0 0 650 400" preserveAspectRatio="xMaxYMax meet">
```
Comment (dennism1997) [inline src/index.html]: Can't we put this inside another file and load it into the page?

File: src/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+@font-face {
```
Comment (dennism1997) [inline src/styles/main.css]: duplicate file

File: src/index.html (hunk 1 above)
Comment (Ishadijcks) [inline src/index.html]: Not sure, but to improve loading times we want to load the smallest number of files possible.
So we could do it, and have gulp compile it to a single `html` file.

File: src/styles/main.css (hunk 2 above)
Comment (Ishadijcks) [inline src/styles/main.css]: Ah I see, webstorm put the `main.ss` inside the `main.less`

**Comment by Ishadijcks** [comment]
pr outdated because of other merge to develop.

**Comment by Ishadijcks** [comment]
Fixed

**Comment by Ishadijcks** [comment]
Ugh, noticed a bug in alreadyCaughtShiny

File: src/scripts/Player.ts [hunk 3]
```diff
@@ -122,7 +124,17 @@ class Player {
...
+    public static alreadyCaughtPokemonShiny(pokemonName: string) {
+        for (let i: number = 0; i < this.caughtPokemonList.length; i++) {
+            if (this.caughtShinyList()[i] == pokemonName) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static capturePokemon(id: number, pokemonName: string, shiny: boolean = false) {
```
Comment (dennism1997) [inline src/scripts/Player.ts]: id is not necessary anymore

Comment (dennism1997) [inline src/scripts/Player.ts]: and how did I make this yellow lol

Comment (Ishadijcks) [inline src/scripts/Player.ts]: Fixed

**Comment by dennism1997** [comment]
we should rotate the route numbers too. Isn't as simple as adding rotate(-90, 0,0) unfortunately

**Comment by Aegyo** [comment]
In calculateRouteCssClass, the GameConstants.RouteCssClass parts don't need to be observable because they never change. What is the purpose of using an enum for this? Just returning the string seems simpler, and you have needed to type out the string anyway to get it from GameConstants.

**Comment by Ishadijcks** [comment]
Yeah good point, I guess no need to make them an enum.

**Pull request merged**